sphinx_rtd_theme/body.html

```diff
@@ -0,0 +1,64 @@
+{% block extrabody %} {% endblock %}
```

Comment:
Rename to body.html

Reply:
I don't understand what do you want to rename.